.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,14 +3,19 @@
 **Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
 Fixes #
 
+<!--
+**Is the PR title following semantic convention?**
+Please refer to [Semantic Pull Requests](https://github.com/zeke/semantic-pull-requests) to view accepted title convention to satisfy this status check.  
```

Comment:
this one seems a little confusing, should we link to https://github.com/open-policy-agent/gatekeeper/blob/master/.github/semantic.yml instead?